crates/sp-domains-fraud-proof/src/verification.rs

```diff
@@ -521,7 +521,8 @@ where
         return Err(InvalidBundleEquivocationError::MismatchedOperatorAndDomain);
     }
 
-    let consensus_block_hash = header_1.header.proof_of_election.consensus_block_hash;
+    // let consensus_block_hash = header_1.header.proof_of_election.consensus_block_hash;
```

Comment:
Such things must have TODOs on them explaining why code is commented-out and when/why it can/should be restored

Reply:
I have already marked a TODO at the source definition of the field. Would you prefer me adding it every commented point ?

Reply:
Generally yes